config/v1/types_cluster_version.go

```diff
@@ -780,17 +803,30 @@ type ConditionalUpdate struct {
 	// +required
 	Release Release `json:"release"`
 
+	// riskNames represents names of ClusterVersionStatus.ConditionalUpdateRisks
+	// that are exposed to the release in a ConditionalUpdate.
+	// The cluster-version operator will evaluate these risks and only
+	// accept the update if there is at least one risk and for every risk
+	// it is either not applied to the cluster or considered acceptable
+	// by the cluster administrator.
+	// +kubebuilder:validation:items:MaxLength=256
+	// +kubebuilder:validation:MaxItems=100
+	// +listType=set
+	// +optional
+	RiskNames []string `json:"riskNames"`
+
 	// risks represents the range of issues associated with
 	// updating to the target release. The cluster-version
 	// operator will evaluate all entries, and only recommend the
 	// update if there is at least one entry and all entries
 	// recommend the update.
+	// DEPRECATED: the risks has been deprecated by RiskNames and will be removed in a future release.
 	// +kubebuilder:validation:MinItems=1
 	// +patchMergeKey=name
 	// +patchStrategy=merge
 	// +listType=map
 	// +listMapKey=name
-	// +required
+	// +optional
```

Comment:
[We are committed to keeping this API through the life of OCPv4][1], so I don't think we can make it optional, because that might break existing v4 clients that rely on its presence.  What we might be able to do is find a way to collapse multiple risks into one "there are risks, see riskNames for details" risk, so clients consuming the old API have a pointer to the new API without breaking.

[1]: https://github.com/openshift/api/pull/2360#discussion_r2150803404